services/net/filemonitor/FilemonitorAction.cs

```diff
@@ -88,6 +98,129 @@ public FileMonitorAction(IHttpRequestClient httpClient, IApiService api, IKafkaM
         await ImportArticlesAsync(manager, articles);
     }
 
+    /// <summary>
+    /// Retrieve files for this connection from the remote server.
+    /// </summary>
+    /// <param name="manager"></param>
+
+    /// <returns></returns>
+    public async Task FetchFiles(IIngestServiceActionManager manager)
```

Comment:
You don't need to change but at some point I suspect we really should not be copying these files over.  However, we'll need to figure out to best read them and determine if the file has already been parsed.